troubleshooting/tutorials/cloud-firelens-crash-repro-template/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM {core dump image build}
+ADD extra.conf /extra.conf
+ADD parser.conf /parser.conf
```

Comment:
This is referenced only here and vaguely in the runbooks details. If we plan on providing a parser file we should consider adding it similar to extra.conf with a comment to fill in

Reply:
So I realized this and then decided to keep it on purpose, so that if you just run `docker build` without thinking it will fail. 

But yea I guess it still serves as a reminder and is less annoying if I just add a parser.conf in the template...